src/app/dashboard/dashboard.component.css

```diff
@@ -0,0 +1,54 @@
+/* DashboardComponent's private CSS styles */
+
+h2 {
+  text-align: center;
+}
+
+.heroes-menu {
```

Comment:
This  should either be classed as part of the dashboard component (We use [BEM](https://getbem.com/), so `.dashboard__heroes-menu`) or moved out to its own component. Keeping the Block class matching the filename helps to locate code from the browser.